README.md

```diff
@@ -48,12 +42,59 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/tidb/stable) and [release notes](https://docs.pingcap.com/tidb/dev/release-notes).
 
+## Community
+
+You can join these groups and chats to discuss and ask TiDB related questions:
+
+- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
+- [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
+- [Chinese User Group](https://asktug.com)
```

Comment:
It's the Chinese forum, actually not Chinese user group, maybe you mean.

Reply:
I'd like to learn a bit about the background. The [original text](https://github.com/pingcap/tidb/pull/23942/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L79) name AskTUG as "User Group(Chinese)" and TUG means TiDB User Group. Shall we refer AskTUG as Chinese forum consistently?

Reply:
I think we should

Reply:
```suggestion
- [Chinese Forum](https://asktug.com)
```

Reply:
@winkyao addressed.